tools/db_crashtest.py

```diff
@@ -774,11 +775,16 @@ def finalize_and_sanitize(src_params):
     if dest_params.get("create_timestamped_snapshot_one_in", 0) > 0:
         dest_params["txn_write_policy"] = 0
         dest_params["unordered_write"] = 0
-    # For TransactionDB, correctness testing with unsync data loss is currently
-    # compatible with only write committed policy
+    # For TransactionDB, correctness testing with unsync data loss and
+    # GetUpdateSince() are currentl compatible with only write committed policy
     if dest_params.get("use_txn") == 1 and dest_params.get("txn_write_policy") != 0:
         dest_params["sync_fault_injection"] = 0
         dest_params["manual_wal_flush_one_in"] = 0
+        dest_params["get_update_since_one_in"] = 0
+    if dest_params.get("get_update_since_one_in") != 0:
+        # Set very high values to avoid WAL cleanup during `GetUpdatesSince()`
+        dest_params["WAL_ttl_seconds"] = 0xFFFFFFFFFFFFFFFF
+        dest_params["WAL_size_limit_MB"] = 0xFFFFFFFFFFFFFFFF
```

Comment:
Should we still set some limit to avoid running out of space?

Reply:
Tuning down the parameter will make the GetUpdateSinceOneIn fail in more ways. This is more difficult than I anticipate will take a look

Reply:
Fixed - it turns out that we don't have to keep such a high value.